src/mscorlib/shared/System/Random.cs

```diff
@@ -263,10 +246,17 @@ public virtual double NextDouble()
         public virtual void NextBytes(byte[] buffer)
         {
             if (buffer == null) throw new ArgumentNullException(nameof(buffer));
-            Contract.EndContractBlock();
             for (int i = 0; i < buffer.Length; i++)
             {
-                buffer[i] = (byte)(InternalSample() % (Byte.MaxValue + 1));
+                buffer[i] = (byte)InternalSample();
```

Comment:
why did you remove `% (Byte.MaxValue + 1));` ? Although, I'm not sure what it was for.

Reply:
Ah I think it's redundant, the byte cast is enough. Fine.